tests/test-safe-redirect-manager.php

```diff
@@ -31,10 +31,22 @@ public static function setUpBeforeClass() {
 		self::$helpers = new \WP_Irving\Test_Helpers();
 	}
 
+	public function setUp() {
+		parent::setUp();
+
+		$this->object = \SRM_Redirect::factory();
+	}
+
 	/**
 	 * Test relative redirect.
 	 */
 	public function test_relative_to_relative() {
+		if ( ! method_exists( $this->object, 'get_redirect_match' ) ) {
```

Comment:
could you abstract this function so you don't have to keep repeating it for each test method?